src/Service/Twig/DateExtension.php

```diff
@@ -73,6 +91,14 @@ public function diff(Environment $env, $date, $now = null): string
         return '';
     }
 
+    /**
+     * @param string|\DateTimeInterface $date
+     */
+    public function dateTime(Environment $env, $date): string
+    {
+        return twig_date_converter($env, $date, $this->timezone)->format('Y-m-d H:i:s');
```

Comment:
Maybe we can add timezone like GitHub:
![Screenshot from 2020-10-08 08-46-39](https://user-images.githubusercontent.com/8239917/95424392-d7ec2100-0942-11eb-86f0-03fced54deea.png)

```suggestion
        return twig_date_converter($env, $date, $this->timezone)->format('Y-m-d H:i:s e');
```